data-table/table.js

```diff
@@ -149,12 +149,68 @@ function onColumnEntryBoxEnter(event, boxValue) {
      */
 }
 
-function addColumns(numberOfColumns, index) {
-    /**
-     * Adds columns from existing table. Will probably call
-     * createEntryCell(). Maybe should also accept index as an argument?
-     * TODO: Fill this out
-     */
+
+/**
+ * Adds colunms to the table.
+ * @param {object} config       - Table configuration object
+ * @param {Number} numberOfColumns - Number of columns to be added
+ * @returns {undefined}         - Doesn't return anything
+ */
+function addMultipleColumns(config, numberOfColumns) {
+    for(let i = 0; i < numberOfColumns; i++){
+        addSingleColumn(config, i);
+    }
+}
+
+/**
+ * Adds a single colunm to the table
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function addSingleColumn(config){ 
+    let table = document.getElementById(config.tableIds.tableElementId);
+    let numRows = table.rows.length; // get length row right now
+    let numCols = table.rows[0].cells.length;
+    for(let i = 0; i < numRows; i++){
+        if(i == 0){  // head 
+            let newCol = table.rows[i].insertCell(-1);
+            newCol.innerHTML("round " + numCols+1);
+        }
+        else{
+            let txtPanel = document.createElement("input");
+            txtPanel.setAttribute('type', 'text');
+            txtPanel.placeholder = "Value";
+            let newCol = table.rows[i].insertCell(-1);
```

Comment:
Note to @Kha6Shafiei - there is an `insertCell` here too, so after this is merged, you`ll have to add `newCol.classList.add(...)` so that the style is applied to added columns